### PR TITLE
Adds CONTRIBUTING.md for the track

### LINKED ISSUE #7: How to set up a local dev environment
See issue exercism/exercism.io#2092 for an overview of operation welcome contributors.

---

Provide instructions on how to contribute patches to the exercism test suites
and examples: dependencies, running the tests, what gets tested on Travis-CI,
etc.

The [contributing document](https://github.com/exercism/x-api/blob/master/CONTRIBUTING.md)
in the x-api repository describes how all the language tracks are put
together, as well as details about the common metadata, and high-level
information about contributing to existing problems, or adding new problems.

The README here should be language-specific, and can point to the contributing
guide for more context.

From the [OpenHatch guide](http://opensource-events.com/#project-setup):

> Here are common elements of setting up a development environment you’ll want your guide to address:
> 
> **Preparing their computer**
> Make sure they’re familiar with their operating system’s tools, such as the terminal/command prompt. You can do this by linking to a tutorial and asking contributors to make sure they understand it. There are usually great tutorials already out there - OpenHatch’s command line tutorial can be found here.
> If contributors need to set up a virtual environment, access a virtual machine, or download a specific development kit, give them instructions on how to do so.
> List any dependencies needed to run your project, and how to install them. If there are good installation guides for those dependencies, link to them.
> 
> **Downloading the source**
> Give detailed instructions on how to download the source of the project, including common missteps or obstacles.
> 
> **How to view/test changes**
> Give instructions on how to view and test the changes they’ve made. This may vary depending on what they’ve changed, but do your best to cover common changes. This can be as simple as viewing an html document in a browser, but may be more complicated.
> 
> Installation will often differ depending on the operating system of the contributor. You will probably need to create separate instructions in various parts of your guide for Windows, Mac and Linux users. If you only want to support development on a single operating system, make sure that is clear to users, ideally in the top-level documentation.

### PR DESCRIPTION
**Pull request by m-a-ge**
Hopefully resolves #7 

I've tried to keep the CONTRIBUTING.md short

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
+[![Build Status](https://travis-ci.org/exercism/groovy.svg?branch=master)](https://travis-ci.org/exercism/groovy)
+[![GitHub issues](https://img.shields.io/github/issues/exercism/groovy.svg)](https://github.com/exercism/groovy/issues)
+[![GitHub forks](https://img.shields.io/github/forks/exercism/groovy.svg)](https://github.com/exercism/groovy/network)
+[![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://raw.githubusercontent.com/exercism/groovy/master/LICENSE)
 [![Join the chat at https://gitter.im/exercism/groovy](https://badges.gitter.im/exercism/groovy.svg)](https://gitter.im/exercism/groovy?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
 Exercism problems in Groovy
 
 ## Contributing Guide
+If you want to be a part of Excercism as a whole and contribute, please refer to the [Contributing Guide](https://github.com/exercism/docs/blob/master/finding-your-way.md).
```
Comment (kytrinyx) [inline README.md]: s/Excercism/Exercism/

Comment (m-a-ge) [inline README.md]: I'm confused and can't understand where exactly the typo is.
Could you please clarify?

Comment (kytrinyx) [inline README.md]: > If you want to be a part of Excercism 

Exercism is misspelled in that sentence

Comment (m-a-ge) [inline README.md]: OMG, sorry. I'll fixed it, thanks!

Comment (kytrinyx) [inline README.md]: LOL, it's totally fine. My colleagues at work are convinced that I'm evil and only make up names that are impossible to spell. They rely exclusively on auto-correct to type my username.

Comment (m-a-ge) [inline README.md]: Sometimes auto-correct is my only solution 🙂 , but in this case I think I didn't get enough sleep.

At any rate, PR is updated and ready to be merged

**Comment by kytrinyx** [comment]
🌻 🌻 🌻

**Pull request merged**